CONTRIBUTING.md

```diff
@@ -1,5 +1,31 @@
-Syntax coding style guidelines
-==============================
+# How to report issues
+
+Before reporting an issue with GitHub, be sure that:
+* you are using the git version of radare2
+* you are using a clean installation
```

Comment:
I also use that, but I think that's not completely clean, because shlr/capstone will be kept.